include/nbl/system/ISystem.h

```diff
@@ -102,7 +102,7 @@ class ISystem final : public core::IReferenceCounted
         void request_impl(SRequestType& req, FutureType& future, RequestParams&& params)
         {
             req.type = params.type;
-            req.params = std::move(params);
+            req.params = params;
```

Comment:
@sadiuk why did you get rid of the `std::move` ?

Reply:
oops, wasn't intending to. I had some issues with that `=` and forgot to return when resolved.